testsuite/CMakeLists.txt

```diff
@@ -28,8 +28,11 @@ set(PROJECT_SHORTNAME "test")
 if ( ${CMAKE_SOURCE_DIR}/ STREQUAL ${CMAKE_CURRENT_SOURCE_DIR} )
   find_package(Feel++ PATHS $ENV{FEELPP_DIR}/share/feelpp/feel/cmake/modules /usr/share/feelpp/feel/cmake/modules /usr/local/share/feelpp/feel/cmake/modules )
   if(NOT FEELPP_FOUND)
-      message(FATAL_ERROR "Feel++ was not found on your system. Make sure to install it and specify the FEELPP_DIR to reference the installation directory.")
+    message(FATAL_ERROR "Feel++ was not found on your system. Make sure to install it and specify the FEELPP_DIR to reference the installation directory.")
   endif()
+else()
+  feelpp_set_options(Feelpp::feelpp_contrib ${PROJECT_NAME})
+  feelpp_set_options(Feelpp::feelpp ${PROJECT_NAME})
```

Comment:
@vincentchabannes perhaps `feelpp_set_cmake_options` would be a better name. it is probably too complicated to have two lines and we should have only one line

Reply:
is it really necessary to use Feelpp ::feelpp_contrib, the definitions are not propagated in Feelpp::feelpp?

Reply:
it is propagated via `target_link_libraries` as a dependencies but the variables are not set in feelpp. if we do that, then we have duplicates

Reply:
what could be done is hide the call to `feelpp_contrib` and `feelpp` inside a `feelpp_set_cmake_options` or something like that